src/formats/color-mode-attributes.js

```diff
@@ -19,39 +34,80 @@ function colorModeAttributes({dictionary: origDictionary, file, options}) {
         name: `${token.name}-hsl`,
         path: [...token.path, 'hsl']
       })
-      acc.push({...token, value: convertToHSL(token.value), darkValue: convertToHSL(token.darkValue)})
+
+      /**
+       * Creates a new token object adjacent to the original
+       * This takes the raw HSL values and wraps it in a HSL css function
+       */
+      const tokenWithHSL = {...token, value: convertToHSL(token.value)}
+
+      /**
+       * Checks if the token object contains one of the supported modes
+       * If yes; mutate the HSL-specific token object with the converted HSL value for the alternate mode
+       */
+      for (const mode of supportedModes) {
+        if (token.hasOwnProperty(mode)) {
+          tokenWithHSL[mode] = convertToHSL(token[mode])
+        }
+      }
+
+      acc.push(tokenWithHSL)
       return acc
     }
 
     dictionary.allTokens = dictionary.allTokens.reduce(reduceProperties, [])
     dictionary.allProperties = dictionary.allProperties.reduce(reduceProperties, [])
   }
 
-  const selector = options.selector ? options.selector : `:root`
   const {outputReferences} = options
   let {allTokens} = dictionary
 
+  const defaultMode = supportedModes[0]
```

Comment:
Could we move this up just below `supportedModes` to have it together? == Easier to read and understand.

Reply:
Yes definitely, forgot to move it when I hoisted the `supportedModes` 🤦 😄